// medoapi/index.js
const axios = require('axios');

function onlineBot(api, token, game) {

    let req = {
        key: api,
        function: 'online_bot',
        token_bot: token,
      };

    if(game) {

        req = {
            key: api,
            function: 'online_bot',
            token_bot: token,
            game: {
                type: game[0],
                content: game[1],
            }
        }

    }

    axios.post('http://api.medodev.xyz:5171/api-post', req).then(async(res) => {
    console.log(res);
  }).catch(error => {
    console.error(error);
  });

};

function updateBot(api, token, game) {

  let req = {
      key: api,
      function: 'update_bot',
      token_bot: token,
    };

  if(game) {

      req = {
          key: api,
          function: 'update_bot',
          token_bot: token,
          game: {
              type: game[0],
              content: game[1],
          }
      }

  }

  axios.post('http://api.medodev.xyz:5171/api-post', req).then(async(res) => {
  console.log(res);
}).catch(error => {
  console.error(error);
});

};

module.exports = {
    onlineBot,
    updateBot
}
